Library/ViewModels/PostCampaignPledgeRewardsSummaryViewModel.swift

```diff
@@ -48,7 +48,10 @@ public protocol PostCampaignPledgeRewardsSummaryViewModelInputs {
 
 public protocol PostCampaignPledgeRewardsSummaryViewModelOutputs {
   var configurePledgeTotalViewWithData: Signal<PledgeSummaryViewData, Never> { get }
-  var loadRewardsIntoDataSource: Signal<[PostCampaignRewardsSummaryItem], Never> { get }
+  var loadRewardsIntoDataSource: Signal<
+    (headerData: PledgeExpandableHeaderRewardCellData?, rewards: [PledgeExpandableHeaderRewardCellData]),
```

Comment:
Nit: If `PledgeExpandableHeaderRewardCellData` is the data type that's used for every cell, including rewards cells, can we please rename it?